classes/UpgradeContainer.php

```diff
@@ -163,7 +163,7 @@ class UpgradeContainer
 
     public function __construct($psRootDir, $adminDir, $moduleSubDir = 'autoupgrade')
     {
-        $this->autoupgradeWorkDir = $adminDir . DIRECTORY_SEPARATOR . $moduleSubDir;
+        $this->autoupgradeWorkDir = $adminDir . '/' . $moduleSubDir;
```

Comment:
Why changing this ?

Reply:
There are many articles and benchmark saying using DIRECTORY_SEPARATOR is useless, it's an old const for very old windows system.

Reply:
Thanks for the answer. I didn't know ;)